digital-guardbook-ui/src/components/logBook/LogBookEntryList.vue

```diff
@@ -1,35 +1,37 @@
 <script setup lang="ts">
-import { useQuery } from '@vue/apollo-composable'
-import gql from 'graphql-tag'
 import { DateTime } from 'luxon';
-import type { LogBookEntry } from './logBookEntry'
+import { useLogBookStore } from '@/store/logBook'
+import { storeToRefs } from 'pinia'
 
 import { useI18n } from 'vue-i18n'
+import { onMounted } from 'vue';
+import { vModelCheckbox } from 'vue';
 
-const {t,n,d} = useI18n({
-  useScope: 'global'
-})
+const store = useLogBookStore()
+
+const { logBookEntries, loading} = storeToRefs(store)
+
+const socket = new WebSocket("ws://localhost:5282/ws");
```

Comment:
Please move the refresh logic into the pinia logbookstore

Reply:
@pwueje can you please check solution?
I've run into the issue, that I don't know the date in the logbookstore

Reply:
hm, I guess your solution won't work with multiple clients, that require different dates. I'll think a bit about the problem. Maybe a solution comes to my mind.